pkg/views/initial/view.go

```diff
@@ -65,6 +65,7 @@ type CommandView struct {
 }
 
 var commandViews []CommandView = []CommandView{
+	{Command: "server", Name: "daytona server", Desc: "(starts the Daytona server in daemon mode)"},
```

Comment:
Please change this to "start the Daytona Server daemon" in order to align with the existing help description

Reply:
done